integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -317,43 +387,28 @@ def export(self, **kwargs):
                     )))
                 continue
 
-            # For instructor-paced and non-audit courses, let the certificate determine course completion
-            if course_details.pacing == 'instructor' and not is_audit_enrollment:
-                completed_date_from_api, grade_from_api, is_passing_from_api, grade_percent = \
-                    self._collect_certificate_data(enterprise_enrollment, channel_name)
-                LOGGER.info(generate_formatted_log(
-                    channel_name, enterprise_customer_uuid, lms_user_id, course_id,
-                    '_collect_certificate_data finished with CompletedDate: {completed_date},'
-                    ' Grade: {grade}, IsPassing: {is_passing},'
-                    .format(
-                        completed_date=completed_date_from_api,
-                        grade=grade_from_api,
-                        is_passing=is_passing_from_api
-                    )))
-            # For self-paced courses, check the Grades API
-            else:
-                completed_date_from_api, grade_from_api, is_passing_from_api, grade_percent = \
-                    self._collect_grades_data(enterprise_enrollment, course_details, is_audit_enrollment, channel_name)
-                LOGGER.info(generate_formatted_log(
-                    channel_name, enterprise_customer_uuid, lms_user_id, course_id,
-                    '_collect_grades_data finished with CompletedDate: {completed_date},'
-                    ' Grade: {grade}, IsPassing: {is_passing},'
-                    .format(
-                        completed_date=completed_date_from_api,
-                        grade=grade_from_api,
-                        is_passing=is_passing_from_api
-                    )))
+            # For audit courses, check if 100% completed
+            # which we define as: no non-gated content is remaining
+            incomplete_count = self.get_incomplete_content_count(enterprise_enrollment, channel_name)
+
+            completed_date_from_api, grade_from_api, is_passing_from_api, grade_percent = self.get_grades_summary(
```

Comment:
maybe add a comment here on how this collects data from only verified enrollments?

Reply:
I did add docstring to the function itself, but this function will also return data for audit mode, just that it won't call the cert api, as of now so not sure what I can write here? but I did add to the docstring of `get_grades_summary`